build-tools/packages/build-tools/src/bumpVersion/showVersions.ts

```diff
@@ -11,10 +11,10 @@ import { Package } from "../common/npmPackage";
 import * as semver from "semver";
 import { strict as assert } from "assert";
 
-export async function showVersions(context: Context, name: string, publishedVersion?: semver.SemVer) {
+export async function showVersions(context: Context, releaseGroup: MonoRepoKind | string, publishedVersion?: semver.SemVer) {
```

Comment:
Might be worth adding docs to this since it's exported

Reply:
It's not actually used yet, so I'll remove the export for now.